src/sentry/incidents/handlers/condition/anomaly_detection_handler.py

```diff
@@ -0,0 +1,94 @@
+import logging
+from typing import Any
+
+from django.conf import settings
+
+from sentry.net.http import connection_from_url
+from sentry.seer.anomaly_detection.get_anomaly_data import get_anomaly_data_from_seer
+from sentry.seer.anomaly_detection.types import (
+    AnomalyDetectionSeasonality,
+    AnomalyDetectionSensitivity,
+    AnomalyDetectionThresholdType,
+    AnomalyType,
+)
+from sentry.snuba.models import QuerySubscription
+from sentry.workflow_engine.models import Condition, DataPacket, DataSource
+from sentry.workflow_engine.registry import condition_handler_registry
+from sentry.workflow_engine.types import DataConditionHandler, DetectorPriorityLevel
+
+logger = logging.getLogger(__name__)
+
+SEER_ANOMALY_DETECTION_CONNECTION_POOL = connection_from_url(
+    settings.SEER_ANOMALY_DETECTION_URL,
+    timeout=settings.SEER_ANOMALY_DETECTION_TIMEOUT,
+)
+
+
+# placeholder until we create this in the workflow engine model
+class DetectorError(Exception):
+    pass
+
+
+@condition_handler_registry.register(Condition.ANOMALY_DETECTION)
+class AnomalyDetectionHandler(DataConditionHandler[DataPacket]):
+    group = DataConditionHandler.Group.DETECTOR_TRIGGER
+    comparison_json_schema = {
+        "type": "object",
+        "properties": {
+            "sensitivity": {
+                "type": "string",
+                "enum": [sensitivity.value for sensitivity in AnomalyDetectionSensitivity],
```

Comment:
if you make `AnomalyDetectionSensitivity` a `StrEnum`, you can do `"enum": [*AnomalyDetectionSensitivity]` .

for keys you want to be a subset of a enum, if you make them `StrEnum` you can do `"enum": [AnomalyType.HIGH_CONFIDENCE, AnomalyType.LOW_CONFIDENCE]`